### PR TITLE
DOCS-11909-ftps-troubleshooting-guide-kt

### PR DESCRIPTION
**Pull request by kevintroller**
- FTPS Troubleshooting guide
- Adding Troubleshooting SDK Components xref link
- Wait merging of https://github.com/mulesoft/docs-mule-sdk/pull/163/files to merge current PR.

### CODE REVIEW

File: ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application's project name.
+. Open the `src/main/resources` path folder.
+. Open the `log4j2.xml` file inside the folder.
+. If the following line is already in the `log4j2.xml` file, uncomment it to enable it; otherwise, add it:
++
+[source,xml,linenums]
+----
+    <AsyncLogger name=“org.mule.extension.ftps” level=“TRACE”/>
+    <AsyncLogger name=“com.jcraft.jsch” level=“DEBUG”/>
```
Comment (giss-ignacio) [inline ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc]: jsch library is not used on the FTPS connector. Instead, apache.commons.net is used through its dependency on the FTP connector
`<AsyncLogger name=“org.mule.extension.ftp” level=“TRACE”/>`
`<AsyncLogger name=“org.apache.commons.net” level=“DEBUG”/>`
can be added to the logging.

**Review by giss-ignacio — CHANGES_REQUESTED** [review]

File: ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
++
+ `-M-Djavax.net.debug=ssl,handshake`
+
+* Try to minimize or remove unnecessary DataWeave data transformations (you could even use hard coded data) to minimize app's complexity and determine whether the issue is with the interaction between the app and the FTPS server.
+
+To get further information about enabling verbose logging for SDK components in your Mule application, see the xref:mule-sdk::troubleshooting.adoc[troubleshoot SDK components] documentation.
+
+== See Also
+* https://help.mulesoft.com[MuleSoft Help Center]
+* xref:sftp-documentation.adoc[SFTP Connector Reference]
```
Comment (giss-ignacio) [inline ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc]: `* xref:ftps-documentation.adoc[FTPS Connector Reference]`

**Review by giss-ignacio — CHANGES_REQUESTED** [review]

File: ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc (hunk 2 above)
Comment (giss-ignacio) [reply ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc]: Please remove the link to the SFTP Connector Reference.

File: ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+----
+  <AsyncLogger name=“org.mule.extension.ftp” level=“TRACE”/>
+  <AsyncLogger name=“org.apache.commons.net” level=“DEBUG”/>
+----
+[start=6]
+. Add the following argument to your run configuration:
++
+ `-M-Djavax.net.debug=ssl,handshake`
+
+* Try to minimize or remove unnecessary DataWeave data transformations (you could even use hard coded data) to minimize app's complexity and determine whether the issue is with the interaction between the app and the FTPS server.
```
Comment (tasmoore) [inline ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc]: Change "minimize app's complexity" to "minimize the app's complexity"

File: ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+To get further information about enabling verbose logging for SDK components in your Mule application, see the xref:mule-sdk::troubleshooting.adoc[troubleshoot SDK components] documentation.
```
Comment (tasmoore) [inline ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc]: Consider changing "To get further information about" to "For additional information about"

File: ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc [hunk 5]
```diff
@@ -0,0 +1,39 @@
+= Troubleshoot FTPS Connector - Mule 4
+
+To troubleshoot Anypoint Connector for FTPs (FTPS Connector), become familiar with the information about troubleshooting Mule application issues
```
Comment (tasmoore) [inline ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc]: Add a period at the end.

**Review by tasmoore — DISMISSED** [review]

File: ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+== Troubleshoot Mule Application Issues
+
+Sometimes you are not sure why the application is failing and you require a better understanding of what is going on in the app's interaction with the FTPS server.
+
+To resolve this error:
+
+* Use the configuration file to increase the logging level on the server side.
+
+* Try to replicate the issue using a FTPS client, for example, Cyberduck or Filezilla. If the issue also occurs with these clients, then it is likely that the issue is not related to the Mule app.
```
Comment (tasmoore) [inline ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc]: Change "a FTPS client" to "an FTPS client"

**Review by tasmoore — DISMISSED** [review]

**Review by tasmoore — APPROVED** [review]

**Review by giss-ignacio — APPROVED** [review]

**Pull request merged**